portality/crosswalks/article_crossref_xml.py

```diff
@@ -273,9 +273,16 @@ def crosswalk_article(self, record, journal):
         if contribs is not None:
             for ctb in contribs:
                 if ctb.attrib["contributor_role"] == 'author':
-                    name = _element(ctb, "x:surname", NS)
-                    name = name + ', ' + _element(ctb, "x:given_name", NS)
-                    bibjson.add_author(name, affiliation=None)
+                    e = _element(ctb, "x:surname", NS)
+                    name = e if e else ""
```

Comment:
I suppose the schema will prevent the case where surname is empty, so the `if e` ... might not be necessary, but there's nothing wrong with coding defensively

Reply:
True, unnecessary. I will clean it up.